VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTopic.java

```diff
@@ -106,4 +115,12 @@ private void appendTopicIds(List<String> ids) {
             child.appendTopicIds(ids);
         }
     }
+
+    public boolean isAllType() {
+        return DiscussionTopic.ALL_TOPICS_ID.equals(identifier);
+    }
+
+    public boolean isFollowingType() {
```

Comment:
Nit: `isAFollowedTopic`? 

Even though the logic for "isAllType" and "isFollowingType" is similar, semantically I feel they could be different to feel more readable. What do you think?